README.md

```diff
@@ -503,7 +503,9 @@ store if `localStorage` is available.
 
 [The Cookie store](http://ember-simple-auth.com/api/classes/CookieStore.html)
 stores its data in a cookie. This is used by the adaptive store if
-`localStorage` is not available.
+`localStorage` is not available. __This store must be used when the
+application uses
+[FastBoot](https://github.com/ember-fastboot/ember-cli-fastboot).__
```

Comment:
That should probably be an assertion instead so that the app crashes in FastBoot as it wouldn't be going to work correctly anyway.